api/app/views/mno_enterprise/jpi/v1/admin/organizations/_main_address_attributes.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.main_address_attributes do
```

Comment:
Should we name this `main_address` and return a `main_address` hash in the JSON (so with the `_attributes` prefix)?

Or is to be consistent with what we sent to mnoe?

Reply:
This was just for consistency. Should I change?